File: test/fixtures/output/ocaml/cohttp/compression.ml
open Cohttp_lwt_unix
open Cohttp
open Lwt

let uri = Uri.of_string "http://mockbin.com/har" in
let headers = Header.add (Header.init ()) "accept-encoding" "deflate, gzip, br" in

Client.call ~headers `GET uri
>>= fun (res, body_stream) ->
  (* Do stuff with the result *)

File: test/fixtures/output/node/unirest/compression.js
const unirest = require("unirest");

const req = unirest("GET", "http://mockbin.com/har");

req.headers({
  "accept-encoding": "deflate, gzip, br"
});

req.end(function (res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});